src/testing/cluster.zig

```diff
@@ -77,9 +78,12 @@ pub fn ClusterType(comptime StateMachineType: fn (comptime Storage: type, compti
         network: *Network,
         storages: []Storage,
         storage_fault_atlas: *StorageFaultAtlas,
+        /// NB: includes both active replicas and standbys. 
         replicas: []Replica,
```

Comment:
Torn a bit about this one. We _can_ rename this and related fields/functions to `nodes_`, but, as the type here is `Replica`, that looks ugly. And I don't want to rename `Replica` to `Node`.